libs/ui/src/histoire.setup.ts

```diff
@@ -1 +1,28 @@
 import './histoire.scss'
+
+import { defineSetupVue3 } from '@histoire/plugin-vue'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { Neo } from './'
+
+export const setupVue3 = defineSetupVue3(({ app }) => {
+  const script = document.createElement('script')
+  script.src = 'https://kit.fontawesome.com/54f29b7997.js'
```

Comment:
move to plugins?

Reply:
nvm I thought it was a good pratice in vue3 to have plugin folder.

Will it load the whole fa kit each time we load a component in kodadot?